resources/sdk/clisdkclient/templates/restclient.mustache

```diff
@@ -243,6 +283,93 @@ func authorize(c config.Configuration) (models.OAuthTokenData, error) {
 	return *oAuthTokenResponse, nil
 }
 
+// Starts the local server at the redirect URI
+func startLocalServerFunc(c config.Configuration, authChannel chan models.OAuthToken) {
+	address := strings.Replace(c.RedirectURI(), "http://localhost", "", -1)
```

Comment:
This code may need to be updated if the redirect URI is stored as `https://localhost`